src/OnboardingSPA/steps/SiteGen/Experience/index.js

```diff
@@ -48,25 +44,27 @@ const SiteGenExperience = () => {
 
 	const checkAndNavigate = ( idx ) => {
 		// 0 - Not Selected
-		// 1-2 Options
+		// 1-3 Options
 		// -1 Skip
 		setSelection( idx );
 		currentData.sitegen.experience.level = idx;
 		setCurrentOnboardingData( currentData );
-		if ( nextStep ) {
-			navigate( nextStep.path );
-		}
 	};
 
 	return (
 		<SitegenAiStateHandler>
 			<CommonLayout isCentered>
 				<div className={ 'nfd-sg-experience-level' }>
-					<SiteGenLoader />
+					<SiteGenLoader
+						watcher={ selection !== 0 ? true : false }
```

Comment:
I don't think we should autonavigate if a selection was not manually made by the user, i.e., via a click event.

Reply:
Let me remove that then.